package-lock.json

```diff
@@ -0,0 +1,905 @@
+{
```

Comment:
Is there any reason to commit this file?
It seems redundant with `yarn.lock`.

Reply:
No reason, good catch!